challenge3/webapp/y-shinada/app/models/house.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: houses
+#
+#  id            :bigint(8)        unsigned, not null, primary key
+#  firstname     :string(255)      not null
+#  lastname      :string(255)      not null
+#  city          :string(255)      not null
+#  num_of_people :integer          default(1), unsigned, not null
+#  has_child     :boolean          default(FALSE), not null
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+# Indexes
+#
+#  index_houses_on_city  (city)
+#
+
+class House < ApplicationRecord
+  scope :with_city, ->(c = "") { !c.blank? ? where(city: c) : self } # rubocop:disable Style/TernaryParentheses
+  scope :with_num_of_people, ->(n = 0) { (n.to_i > 0) ? where(num_of_people: n) : self }
```

Comment:
このままでまったく問題ないが、Rails には positive? というメソッドがある。

```suggestion
  scope :with_num_of_people, ->(n = 0) { (n.to_i.positive? ? where(num_of_people: n) : self }
```
https://api.rubyonrails.org/classes/Numeric.html#method-i-positive-3F